cli/tests/test_parallelize_command.rs

```diff
@@ -399,9 +433,9 @@ fn test_parallelize_multiple_heads_with_and_without_children() {
         &["parallelize", "description(0)", "description(1)"],
     );
     insta::assert_snapshot!(get_log_output(&test_env, &workspace_path), @r###"
-    @  49fe9e130d15 2
-    ◉  9d0c0750973c 0
-    │ ◉  2047527ade93 1
+    ◉  2047527ade93 1
```

Comment:
nit: It doesn't really matter, but is there any way to preserve the graph order better? I haven't looked at exactly how `jj log` orders the commits, but I'm guessing modification time plays a role?

Reply:
It's not exactly the modification time, or at least not the time recorded in the commit. It's the order they were written to the index. There's not much we can do here without a lot of ugly hacks (like removing these commits from the index and then adding them back).

Reply:
Yeah, not worth it then.